### PR TITLE
Javascript javascript3 week3

### PR DESCRIPTION
**Pull request by sergii-borodin**
Homework done. Waiting for feedback. Thanks in advance. Sorry to be late

### CODE REVIEW

File: javascript/javascript3/week3/canvas/main.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    context.beginPath();
+    context.fillStyle = "#" + Math.floor(Math.random() * 16777215).toString(16);
+
+    context.arc(
+      this.x,
+      this.y,
+      this.r,
+      this.startAngle,
+      this.endAngle,
+      this.fillColor
```
Comment (dsamourkasidis) [inline javascript/javascript3/week3/canvas/main.js]: The fillColor parameters is not supported by the arc function: https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/arc

Instead, you could use the fillColor parameter in the ``` context.fillStyle```

File: javascript/javascript3/week3/Shopping_cart/main.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      EUR: 7.75,
+    };
+    const currentProductPrice = this.price;
+    for (const key in currencyRate) {
+      if (key === currency) {
+        if (Object.hasOwnProperty.call(currencyRate, key)) {
+          const currentCurrencyRate = currencyRate[key];
+          const productPrice = currentProductPrice / currentCurrencyRate;
+          return productPrice;
+        }
```
Comment (dsamourkasidis) [inline javascript/javascript3/week3/Shopping_cart/main.js:21]: Works perfectly fine but just to consider a shorter way to do this:
```
if (currency in Object.keys(currencyRate){
          const currentCurrencyRate = currencyRate[currency];
          const productPrice = currentProductPrice / currentCurrencyRate;
          return productPrice;
}
```

File: javascript/javascript3/week3/Shopping_cart/main.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
+const body = document.querySelector("body");
+
+class Product {
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
+  convertToCurrency(currency) {
```
Comment (dsamourkasidis) [inline javascript/javascript3/week3/Shopping_cart/main.js:8]: Just for a future improvement you could try using an existing api to do the conversion. For instance this:
https://free.currencyconverterapi.com/

**Review by dsamourkasidis — APPROVED** [review]
Great work here Sergii :100: 
Nothing much to add really, just a few minor comments for future improvement :)

File: javascript/javascript3/week3/Shopping_cart/main.js (hunk 3 above)
Comment (sergii-borodin) [reply javascript/javascript3/week3/Shopping_cart/main.js:8]: Thanks a lot Dimitris for your review. It's helpfull. Have a nice week.

**Pull request merged**